METADATA.yaml

```diff
@@ -1,37 +1,25 @@
-title: User Interface & Accessibility
-description: wg-ui-and-a11y Working Group is responsible for AMP's visual components & interactions and AMP's overall accessibility and user experience (including published guidelines for AMP UX). The UI Working Group also has overall responsibility for ensuring AMP is accessible, though each WG is responsible for ensuring accessibility within their areas of responsibility.
+title: Components
+merged: 
+ - ui-and-a11y
+description: wg-components Working Group is responsible for AMP components, the overall health of AMP Pages, analytics features, and integrations with partner technologies. For overall accessibility, this group is also the first point of contact, however each WG is responsbile for ensuring accessibility within their areas of responsibility.
 facilitator:
-  login: nainar
-  name: Naina Raisinghani
+  login: zhouyx
+  name: Yuxuan Zhou
```

Comment:
Yuxuan has left the team. The updated membership can be found at https://github.com/ampproject/wg-components.

Reply:
The facilitator listed for the group at the link above is @micajuine-ho FYI